geometry_msgs/Quaternion orientation
int32 jointType